MIGRATION.md

```diff
@@ -0,0 +1,109 @@
+The steps are valid for classic javascript and typescritp repositories. If you are using esm modules or react the eslint configuration needs to include some more modules. Please see README.md for details.
+
+- [ ] Clone your repository to a local workspace
+- [ ] Verify that current (old) eslint configuration is working
+
+  Execute `npm run lint` and check for errors. Consider fixing all existing errors before starting the migration.
+      
+- [ ] remove packages no longer needed
+
+      npm uninstall eslint-config-prettier
+      npm uninstall eslint-plugin-prettier
+      npm uninstall prettier
+
+- [ ] update eslint to current version
+
+      npm i eslint@latest --save-dev
+
+- [ ] install iobroker standard rules
+
+      npm i @iobroker/eslint-config --save-dev
+
+- [ ] remove old configuration files
+
+      .eslintignore
+      .eslintrc.json
+      .prettierignore
+      .prettierrc.js
+      
+- [ ] create new configuration file 'eslint.config.mjs' for eslint
+
+      //  
+      // ioBroker eslint template configuration file for js and ts files
+      // Please note that esm or react based modules need additional modules loaded.
+      //  
+      
+      import config from '@iobroker/eslint-config';
+    
+      export default [
+          ...config,
+      
+          {
+              // specify files to exclude from linting here
+              ignores: [
+                  '*.test.js', 
+                  'test/**/*.js', 
+                  '*.config.mjs', 
+                  'build', 
+                  'admin/build', 
+                  'admin/words.js',
+                  'admin/admin.d.ts',
+                  '**/adapter-config.d.ts'     
+              ] 
+          },
+      
+          {
+              // you may disable some 'jsdoc' warnings - but using jsdoc is highly recommended
+              // as this improves maintainability. jsdoc warnings will not block buiuld process.
+              rules: {
+                  // 'jsdoc/require-jsdoc': 'off',
+              },
+          },
+          
+      ];
+  
+- [ ] create new configuration file 'prettier.config.mjs' for prettier
+
+      //  
+      // iobroker prettier configuration file
+      //  
+      
+      import prettierConfig from '@iobroker/eslint-config/prettier.config.mjs';
+      
+      export default {
+          ...prettierConfig,
+      
+          // uncomment next line if you prefer double quotes
+          //singleQuote: false,
+      }
+  
+- [ ] check and eventually adapt script definition at pacakge.json
+   
+  Your 'lint' script definitioon at io-package.json should read like this
+
+      {
+          "scripts": {
+              "lint": "eslint -c eslint.config.mjs ."
+          }
+      }
+
+- [ ] update .npmignore (if still in use)
+
+  If you still use .npmignore and die not yet switch to use `files` section within opackage.json, add the follwoing files to `.npmignore`
```

Comment:
If you still use `.npmignore` and not yet switched to use `files` section within `package.json`, add the following files to `.npmignore`

Reply:
fixed